CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+## 2.29.0-rc.0 / 2021-07-29
+
+* [CHANGE] Remove experimental tag on `--storage.tsdb.allow-overlapping-blocks` flag. #9117
+* [CHANGE] Remove experimental tag on `--storage.tsdb.retention.size` flag. #8728
+* [FEATURE] Add Kuma service discovery. #8844
+* [FEATURE] Add `__meta_gce_interface_ipv4_<name>` meta label to GCE discovery. #8978
+* [FEATURE] Add `__meta_ec2_availability_zone_id` meta label to EC2 discovery. #8978
+* [FEATURE] Add `__meta_hetzner_hcloud_labelpresent_<labelname>` meta label to Hetzner discovery. #9028
+* [FEATURE] Allow configuring exemplar storage via file and make it reloadable. #8974
+* [FEATURE] promtool: Add feature flags flag `--enable-feature`. #8958
+* [FEATURE] promtool: Add service discovery configuration validation. #8950
+* [ENHANCEMENT] Allow filtering with matchers in label-names API. #9083
```

Comment:
This is not user visible, I would not have it here.

Reply:
https://github.com/prometheus/prometheus/pull/9133#discussion_r678993835 I think I'd want to include it for API users.

Reply:
And to be clear, this only changes the Go interface with some internal enhancements, while the http API behaviour is same as before (we support the matchers in http API for label names already)

Reply:
Ooooooh, I didn't realize this wasn't added to the HTTP API. Thanks for clarifying, yes then it shouldn't be included.